content/en/docs/reference/glossary/event.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Event
+id: event
+date: 2022-01-16
+full_link: /docs/reference/kubernetes-api/cluster-resources/event-v1/
+short_description: >
+   A report of an event somewhere in the cluster. It generally denotes some state change in the system.
+aka: 
+tags:
+- core-object
+- fundamental
+---
+Each Event is a report of an event somewhere in the {{< glossary_tooltip text="cluster" term_id="cluster" >}}. 
```

Comment:
```suggestion
Each `event` is a report of an event somewhere in the {{< glossary_tooltip text="cluster" term_id="cluster" >}}. 
```
@sftim @anubha-v-ardhan Would you agree on doing this?

Reply:
Other terms in K8s glossary don't have this. So I think we shouldn't do it to keep it all uniform. Also, it doesn't make any significant difference as such in my opinion. Thoughts?

Reply:
Other terms in K8s glossary don't have this so I think we shouldn't to maintain uniformity.
Also backticks are generally done to enclose code segments. Writing the word just as is works fine in my opinion. Thoughts?

Reply:
Okay, agree on backticks!
But I don't think capitalizing `Event` makes sense, not sure!
@sftim What do you say?

Reply:
Yes!! Capital "E" must have slipped in as "Each" was added later by a prior suggestion. Thank you for pointing out.